### PR TITLE
Update code to utilize namespacing with statuses

### PR DESCRIPTION
**Pull request by whyderrick**
The last PR changed the names on some model constants but didn't
update those references throughout the code. This updates that
oversight.

### CODE REVIEW

File: spec/features/submission_spec.rb [hunk 1]
```diff
@@ -26,11 +26,40 @@
...
+      submit.form
+
+      expect(page).to have_flash_message(
+        :notice,
+        text: t("flash.actions.create.notice"),
+      )
+      expect(page).to have_text(submission_title)
+      expect(page).to have_text(new_poem_title)
+    end
+  end
```
Comment (gylaz) [inline spec/features/submission_spec.rb]: Is this test relevant to this commit/PR? If so, I'm not sure how.

**Review by gylaz — APPROVED** [review]
One question.

File: spec/features/submission_spec.rb (hunk 1 above)
Comment (whyderrick) [reply spec/features/submission_spec.rb]: It's not. Thanks. I wasn't careful enough when breaking this out into its own PR. I'll update it.
